src/main/java/thestonedturtle/partypanel/ui/PlayerBanner.java

```diff
@@ -122,20 +125,23 @@ public void recreatePanel()
 		nameContainer.setOpaque(false);
 
 		final JLabel usernameLabel = new JLabel();
+		usernameLabel.setLayout(new OverlayLayout(usernameLabel));
 		usernameLabel.setHorizontalTextPosition(JLabel.LEFT);
 		if (player.getUsername() == null)
 		{
 			usernameLabel.setText("Not logged in");
 		}
 		else
 		{
-			final String levelText = player.getStats() == null ? "" : " (Lvl - " + player.getStats().getCombatLevel() + ")";
+			final String levelText = player.getStats() == null ? "" : " (Lvl " + player.getStats().getCombatLevel() + ")";
```

Comment:
We may want to change this to `(Level-XXX)` to match the right-click display of the user but if space is an issue this is fine

Reply:
Yes, space was an issue. I tested some long RSNs. But I think `(lvl-123)` might match a little better so I've changed it to that for now.